staging/src/k8s.io/apimachinery/pkg/conversion/converter_test.go

```diff
@@ -111,72 +111,33 @@ func TestConverter_byteSlice(t *testing.T) {
 func TestConverter_MismatchedTypes(t *testing.T) {
 	c := NewConverter(DefaultNameFunc)
 
-	err := c.RegisterConversionFunc(
-		func(in *[]string, out *int, s Scope) error {
-			if str, err := strconv.Atoi((*in)[0]); err != nil {
-				return err
-			} else {
-				*out = str
-				return nil
-			}
+	convertFn := func(in *[]string, out *int, s Scope) error {
+		if str, err := strconv.Atoi((*in)[0]); err != nil {
+			return err
+		} else {
+			*out = str
+			return nil
+		}
+	}
+	if err := c.RegisterUntypedConversionFunc(
+		(*[]string)(nil), (*int)(nil),
+		func (a, b interface{}, s Scope) error {
+			return convertFn(a.(*[]string), b.(*int), s)
 		},
-	)
-	if err != nil {
+	); err != nil {
 		t.Fatalf("Unexpected error: %v", err)
 	}
 
 	src := []string{"5"}
 	var dest *int
-	err = c.Convert(&src, &dest, 0, nil)
-	if err != nil {
+	if err := c.Convert(&src, &dest, 0, nil); err != nil {
 		t.Fatalf("unexpected error: %v", err)
 	}
 	if e, a := 5, *dest; e != a {
 		t.Errorf("expected %#v, got %#v", e, a)
 	}
 }
 
-func TestConverter_DefaultConvert(t *testing.T) {
-	type A struct {
-		Foo string
-		Baz int
-	}
-	type B struct {
-		Bar string
-		Baz int
-	}
-	c := NewConverter(DefaultNameFunc)
-	c.Debug = testLogger(t)
-	c.nameFunc = func(t reflect.Type) string { return "MyType" }
-
-	// Ensure conversion funcs can call DefaultConvert to get default behavior,
-	// then fixup remaining fields manually
```

Comment:
This is (and already was) no longer true - the new-style conversion no longer can call DefaultConvert.
It seems DefaultConvert is not used anywhere, so I'm going to remove it in the subsequent PR.
@smarterclayton